package.json

```diff
@@ -34,7 +34,7 @@
     "presnap": "npm run prepare",
     "test": "tap",
     "snap": "tap",
-    "format": "prettier --write . --loglevel warn",
+    "format": "prettier --write . --log-level warn",
```

Comment:
Prettier wanted to make changes to basically every file in the repo, I just formatted test/escape-has-magic.js to keep the diff small. It also complained that the option name has a dash.